recipe-server/client/control/tests/components/test_PreferenceExperimentFields.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import {
+  PreferenceExperimentFields,
+} from 'control/components/action_fields/PreferenceExperimentFields';
+
+describe('<PreferenceExperimentFields>', () => {
+  it('should render without errors', () => {
+    const wrapper = () => {
+      shallow(<PreferenceExperimentFields preferenceBranchType="default" />);
+    };
+
+    expect(wrapper).not.toThrow();
+  });
+
+  it('should render a warning if using the user preference branch type', () => {
+    const wrapper = shallow(<PreferenceExperimentFields preferenceBranchType="user" />);
+    const warning = wrapper.find('.field-warning');
```

Comment:
re: adding the warning as a static class member - I _thiiiink_ we could do something like `wrapper.find(PreferenceExperimentFields.userPrefWarning)` here.